experimental/oak_baremetal_channel/src/lib.rs

```diff
@@ -88,6 +96,7 @@ const MAX_FRAME_BODY_SIZE: usize = MAX_FRAME_SIZE - FRAME_HEADER_SIZE;
 #[derive(Clone)]
 struct Frame {
     method_or_status: MethodOrStatus,
+    message_id: MessageId,
```

Comment:
FYI (not specifically to this line), this may be an interesting read for you since you are getting deeper and deeper into these packing and alignment details: http://www.catb.org/esr/structure-packing/ (I haven't read it myself though -- if you do, let me know whether it is actually relevant for us)